workflow-templates/discussion-comment-notice.yml

```
@@ -0,0 +1,13 @@
+on:
+  discussion_comment:
+    types: [created]
+
+jobs:
+  discussion_commented:
+    if: github.event.discussion && github.event.comment
+    uses: route06/actions/.github/workflows/_discussions-comment-notice.yml@v1
```

📝 https://github.com/route06/actions/pull/13 がマージされると使える

---

マージされたものの、ファイルの命名規則が変わったため調整します ✍🏻 

```suggestion
    uses: route06/actions/.github/workflows/gh_discussion_comment_to_slack.yml@v2
```